redux_userlist_06/src/components/Dashboard/userProfile.tsx

```diff
@@ -1,21 +1,37 @@
+import { useEffect, useState } from "react";
 import { Link } from "react-router-dom";
 
 import { UserDetails } from "../../interfaces";
 
 import "./styles.css";
 
-const UserProfile = (props: any)=> {
+const UserProfile = (props: any) => {
   const details: UserDetails = props.data;
```

Comment:
this is not the way to give type to props,
refer this https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/basic_type_example/